packages/viewer/src/modules/SceneObjectManager.js

```diff
@@ -1,378 +1,404 @@
-import * as THREE from 'three'
-import debounce from 'lodash.debounce'
-import SceneObjects from './SceneObjects'
+import * as THREE from 'three';
+import debounce from 'lodash.debounce';
+import SceneObjects from './SceneObjects';
```

Comment:
Make sure you reformat all files back to their original formatting once you remove the prettier config